mcdreforged/info_reactor/impl/server_reactor.py

```diff
@@ -19,6 +19,19 @@ def react(self, info):
 				self.mcdr_server.with_flag(MCDReforgedFlag.SERVER_STARTUP)
 				self.mcdr_server.plugin_manager.dispatch_event(MCDRPluginEvents.SERVER_STARTUP, ())
 
+			version = handler.parse_server_version(info)
+			if version:
+				self.mcdr_server.logger.debug(f'Server version detected: {version}', option=DebugOption.REACTOR)
+				self.mcdr_server.server_info.version = version
+
+			ip = handler.parse_server_ip(info)
+			if ip is not None:
+				self.mcdr_server.logger.debug(f'Server ip detected: {ip[0]}:{ip[1]}', option=DebugOption.REACTOR)
+				try:
+					self.mcdr_server.server_info.set_ip(*ip)
```

Comment:
I prefer to not use unpacking to pass the arguments here, since it's ambiguous

Reply:
forgot to remove execption catching rue

Reply:
removed